src/SDKs/Kusto/Management.Kusto/Generated/SdkInfo_KustoManagementClient.cs

```diff
@@ -1,41 +0,0 @@
-
-// <auto-generated>
-// Code generated by Microsoft (R) AutoRest Code Generator.
-// Changes may cause incorrect behavior and will be lost if the code is
```

Comment:
This file should not be deleted. Was the code generated using generate.ps1 script?